xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -278,16 +278,16 @@ public void createSingleUseToken(final Card card, final int amount, final TokenC
      * @param tokenCallback The callback that will be called when the token creation completes or
      *                      fails
      */
-    public void createMultipleUseToken(final Card card, final TokenCallback tokenCallback) {
-        createSingleOrMultipleUseToken(card, "0", true, tokenCallback);
+    public void createMultipleUseToken(final Card card, final boolean shouldAuthenticate, final TokenCallback tokenCallback) {
+        createSingleOrMultipleUseToken(card, "0", shouldAuthenticate, true, tokenCallback);
     }
 
     @Deprecated
-    public void createToken(final Card card, final String amount, final boolean isMultipleUse, final TokenCallback tokenCallback) {
-        createSingleOrMultipleUseToken(card, amount, isMultipleUse, tokenCallback);
+    public void createToken(final Card card, final String amount, final boolean shouldAuthenticate, final boolean isMultipleUse, final TokenCallback tokenCallback) {
```

Comment:
This is a deprecated method. Should be left as is so it does not break customers. It will be deleted later on.